product/roadmap.md

```diff
@@ -50,10 +53,50 @@ We are dedicated to maintain support for three different Kubernetes versions wit
 - Provide disaster recovery procedures for Otomi core applications, such as Gitea, Keycloak and Harbor
 - Show a compliance report (code vulnerabilities, image vulnerabilities, security violations, container vulnerabilities, config audit, exposed secrets) per workload
 
-## After Q2 2024
+### After Q2 2024
 
 After Q2 2024 we have the following goals:
 
 - Make Otomi more plugable by enabling users to bring their own platform apps
 - Migrate to ambient mesh using eBPF
 - Remove platform app forms in favor of generated values that can be customized in an editor
+
+## Removed features by release
+
+### v3.0
+
+#### Drone
+
+Otomi will stop delivering Drone as a platform app.
+The Tekton app is introduced in Otomi v2.0 as a replacement. There is no migration procedure, Drone is used primarely as an Otomi deployment pipeline. However if you used it as for custom pipeline then you may need to migrate them Tekton.
+
+:::note
+This app will not be deleted if it is already deployed.
+:::
+
+#### Hashicorp Vault
+
+Otomi will stop delivering the Hashicorp Vault as a platform app.
+The SealedSecrets app is introduced in Otomi v2.7 as a replacement.
+The procedure to migrate from Hashicorp Vault to SealedSecrets is going to be announced soon.
+
+:::note
+This app will not be deleted if it is already deployed.
+The external-secrets app is still going to be part of Otomi
+:::
+
+#### OPA Gatekeeper
+
+Otomi will stop delivering the OPA Gatekeeper as a platform app.
+The Kyverno app is introduced in Otomi v2.7 as a replacement. There is no migration procedure, as the basic policies are already covered. However if you used your custom policies then you may need to migrated them on your own.
```

Comment:
`However if you used your custom policies then you may need to migrated them on your own.` -> `However if you used it for custom policies then you may need to migrate them on your own.`

Reply:
ok